### DISCUSSION TITLE
[Suggestion] provide an children builder

### DISCUSSION THREAD

**Discussion by iulian0512** [darioielardi/flutter_speed_dial#240] [Ideas]
A children builder function that is invoked on speed dial click would allow a custom number of children that depends on the current application state. 
For example the speed dial is used to add 3 categories of items into a list, but in some cases only 2 categories of items are possible to be added that's where the builder function comes in handy, it is invoked and based on some conditions it returns 2 children.

**Comment by prateekmedia** [comment]
Contributions are welcome, the package is only receives security-updates for now.
